docker-compose.yml

```diff
@@ -1,199 +1,174 @@
 version: "3.8"
 
 networks:
-    omotes:
+  omotes:
+    driver: bridge
```

Comment:
This is the default, any reason you are making this explicit? Nothing wrong with it of course.

Reply:
I thought it fixed an issue of having to add the network name in front of the service name to be able to find it. But apparently this was not what solved it.